### ISSUE TITLE
Custom CSS not working in iFrames

### ISSUE THREAD

**Issue by RobertNJr** [ankit/stylebot#189]
iFrame sources aren't being affected by StyleBot. Even if you are to go to the iFrame's source in a new tab, and edit it there.

**Comment by JonasNo** [comment]
I can give an example of nested iframes, in where stylebot do not work.
Example (adblock is a must, nasty ads, can use new profile for safety and privacy):
www dot toonova dot com/justice-league-unlimited-season-3-episode-8

The css you want to edit is the "close ad" dialog box that is over each and every flashplayer.
Do not be fooled the dialog is produced via javascript and html.

You can work around nested iframes by adding the urls of the iframes but an extension should be able to insert css into nested iframes automatically via an option.

I did this myself via a one-off extension a few years ago. It should still be possible in chrome.
Walked the iframe tree, level by level inserting code in each iframe.
The code to do this was quick and dirty.
It worked up until the crappy hybrid hard drive went bad.

**Comment by tobia** [comment]
I'd like to add a style rule to an iframe (Kaltura video player) but I don't really understand how to do it.

I cannot open the iframe in another tab, because it does not have an URL. It's entirely created and populated by (tons of) Javascript.

Here's an example: http://player.kaltura.com/

How would you go about adding something like this to the iframe in the middle of the page?

    .controlBarContainer {
        display: none;
    }

**Comment by RobFreundlich** [comment]
I have a similar issue on all pages that load comments using an iframe.  For example, many sites use Facebook comments, and when I right-click on something in the comment body and select "Stylebot", nothing happens.

When I view frame source, I can see the markup, and I can see that the site is using disqus.  So I manually created the appropriate CSS rule for disqus.com in Stylebot options, but they don't take effect in the comments.  I also tried setting the CSS in my rule for facebook.com, but that doesn't affect the comments either.

One example is http://theinfluence.org/the-surgeon-generals-addiction-report-repeats-old-bromides-heres-what-he-ought-to-be-telling-us/
